kubernetes/kserve/kserve_wrapper/inference_pb2_grpc.py

```diff
@@ -0,0 +1,124 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
can we generate the code on fly (eg during installation) instead of copy/paste the auto-gen code at here?